Cristi's FinOps brain dump: add companies, blogs, newsletters, podcasts

cristim opened the pull request:
- [x] I have read the [contribution guidelines](https://github.com/jmfontaine/awesome-finops/blob/main/CONTRIBUTING.md) and ensured my submission follows it.
- [x] I have searched previous suggestions before submitting this pull request.
- [ ] This pull request contains only one suggestion.

I added a bunch of suggestions in a single commit to reduce overhead, because otherwise it would have been dozens of them.

cristim committed:
Cristi's FinOps brain dump: add companies, blogs, newsletters, podcasts

hugorut reviewed (commented):
hey @cristim thanks for including us! Any reason why you've put @infracost in the "Commercial Tools" section? Our tool is completely free and open source? https://github.com/infracost/infracost

cristim committed:
Add more descriptions, make them shorter

serverlesspolska committed:
Add CloudPouch

cristim committed:
Merge pull request #1 from CloudPouch/main

Add CloudPouch

heidicarson committed:
Update README.md

Added this description under the Commercial Tools section: Pepperdata - Real-time, automated cost optimization for Amazon EMR and Amazon EKS. Tags: automated_optimization, EKS, EMR, Kubernetes, multi_cloud

cristim committed:
Merge pull request #3 from heidicarson/patch-4

Update README.md

CloudFinOpsio committed:
awesome-finops update (#4)

Added CloudThread, FinOut and DigiUsher

---------

Co-authored-by: Cristian Magherusan-Stanciu <cristim@users.noreply.github.com>

k2589 committed:
add perfectscale.io (#5)

Co-authored-by: Karim Shakirov <karimshakirov@Karims-MBP.netis.cc>

sartya committed:
Add NudgeBee to Commercial Tools (#10)
